janusgraph-test/src/test/java/org/janusgraph/core/CQLConfiguredGraphFactoryTest.java

```diff
@@ -0,0 +1,89 @@
+// Copyright 2017 JanusGraph Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.janusgraph.core;
+
+import com.datastax.driver.core.Session;
+import org.apache.commons.configuration.MapConfiguration;
+import org.janusgraph.graphdb.database.StandardJanusGraph;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.CassandraContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.janusgraph.graphdb.configuration.GraphDatabaseConfiguration.GRAPH_NAME;
+import static org.janusgraph.graphdb.configuration.GraphDatabaseConfiguration.STORAGE_BACKEND;
+import static org.janusgraph.graphdb.configuration.GraphDatabaseConfiguration.STORAGE_HOSTS;
+import static org.janusgraph.graphdb.configuration.GraphDatabaseConfiguration.STORAGE_PORT;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+@Testcontainers
+public class CQLConfiguredGraphFactoryTest extends ConfiguredGraphFactoryTest {
+
+    private static final String CASSANDRA_VERSION = "3.11.6";
+    private static final String CASSANDRA_IMAGE = "cassandra";
+
+    @Container
+    public static final CassandraContainer cqlContainer;
```

Comment:
Can you use JanusGraphCassandraContainer if possible?

Reply:
I try to use it when I first added the tests but it seems to not allow multiple storage backends. I added some details about this before [in this conversation](https://github.com/JanusGraph/janusgraph/pull/2148#discussion_r450122533)